packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -120,6 +123,9 @@ async function convertFHIRBundleToMedicalRecord({
     })
     .promise();
 
+  const isSandbox = Config.isSandbox();
+  const patientMatch = patientMatches(patient.data);
+
   // Send it to conversion
   const payload: ConversionInput = {
```

Comment:
This is only used when not sandbox+patientMatch, moving it there keeps the code cohesive and easier to understand.

Also, NABD but since the `if` returns, there's no need for an `else` - slightly easier to read